content/en/docs/setup/install/istioctl/index.md

```diff
@@ -94,7 +94,12 @@ $ istioctl install --set profile=demo
 ## Check what's installed
 
 The `istioctl` command saves the `IstioOperator` CR that was used to install Istio in a copy of the CR named `installed-state`.
-You can inspect this CR if you lose track of what is installed in a cluster.
+If you lose track of what is installed in a cluster, you can inspect this CR.
+For example, dump its content into a YAML file using the following command:
```

Comment:
Maybe a more subtle way to mention getting the resources would be ok?

```suggestion
Instead of inspecting the deployments, pods, services and other resources that were installed by Istio, for example:

{{< text bash >}}
$ kubectl -n istio-system get deploy
NAME                   READY   UP-TO-DATE   AVAILABLE   AGE
istio-ingressgateway   1/1     1            1           49m
istiod                 1/1     1            1           49m
{{< /text >}}

You can inspect the `installed-state` CR, to see what is installed in the cluster, as well as all custom settings.
For example, dump its content into a YAML file using the following command:
```